src/tools/clippy/clippy_lints/src/lib.rs

```diff
@@ -3,7 +3,6 @@
 #![feature(box_patterns)]
 #![feature(drain_filter)]
 #![feature(in_band_lifetimes)]
-#![feature(iter_zip)]
```

Comment:
It will be easier to pull this diff upstream back into Clippy if the changes in the Clippy subtree are a separate commit from the one that diffs the main tree. It can and should stay in this PR, just split into two commits.